hyperspy/io.py

```diff
@@ -478,6 +478,9 @@ def save(filename, signal, overwrite=None, **kwds):
         ensure_directory(filename)
         if overwrite is None:
             overwrite = overwrite_method(filename)
+        if overwrite is False:
```

Comment:
Travis and appveyor are failing because of other tests, which are now fixed in the upstream branch. You can merge upstream/RELEASE_next_patch into your branch if you want to get travis and appveyor passing, but you don't need to worry about this for now!